manifests/init.pp

```diff
@@ -56,9 +56,15 @@
       minute  => '*/2',
     }
   }
-
+  $report_set = false
   if $enable_reports {
-    if $reports != '' {
+    if $reports == unset {
```

Comment:
Is this supposed to be `undef`?

Reply:
So apparently you cant set a string param to undef unless it's optional?

Reply:
It was a typo though, you are correct.